cpgserver/src/test/scala/io/shiftleft/cpgserver/cpg/DummyCpgProviderSpec.scala

```diff
@@ -0,0 +1,39 @@
+package io.shiftleft.cpgserver.cpg
+
+import java.util.UUID
+
+import cats.data.OptionT
+import cats.effect.IO
+
+import io.shiftleft.codepropertygraph.Cpg
+import io.shiftleft.cpgserver.BaseSpec
+import io.shiftleft.cpgserver.model.CpgOperationResult
+
+class DummyCpgProviderSpec extends BaseSpec {
+
+  private def withNewCpgProvider[T](f: DummyCpgProvider => T): T = {
+    f(new DummyCpgProvider)
+  }
+
+  "Creating a CPG" should {
+    "return a UUID referencing the eventual CPG" in withNewCpgProvider { cpgProvider =>
+      noException should be thrownBy cpgProvider.createCpg(Set.empty).unsafeRunSync()
+    }
+  }
+
+  "Retrieving a CPG" should {
+//    "return a success if the CPG was created successfully" in withNewCpgProvider { cpgProvider =>
```

Comment:
These two tests are still commented out. Do they still not be implemented or should they be removed?

Reply:
Ah nice catch - I need to uncomment this.